src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -0,0 +1,513 @@
+| IntendedFor | RECOMMENDED. This can be an MRI/CT or a file containing the operative photo, x-ray or drawing with path relative to the project folder. If only a surface reconstruction is available, this should point to the surface reconstruction file. Note that this file should have the same coordinate system specified in `iEEGCoordinateSystem`. For example, **T1**: `sub-<label>/ses-<label>/anat/sub-01_T1w.nii.gz`  **Surface**: `/derivatives/surfaces/sub-<label>/ses-<label>/anat/sub-01_T1w_pial.R.surf.gii` **Operative photo**: `/sub-<label>/ses-<label>/ieeg/sub-0001_ses-01_acq-photo1_photo.jpg` **Talairach**: `/derivatives/surfaces/sub-Talairach/ses-01/anat/sub-Talairach_T1w_pial.R.surf.gii`                                                                 |
+
+Fields relating to the iEEG electrode positions:
+
+| Field name                          | Definition                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| :---------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| iEEGCoordinateSystem                | REQUIRED. Defines the coordinate system for the iEEG electrodes. For example, “ACPC”. See Appendix VIII: preferred names of Coordinate systems. If "Other" (e.g., individual subject MRI), provide definition of the coordinate system in `[iEEGCoordinateSystemDescription]`. If positions correspond to pixel indices in a 2D image (of either a volume-rendering, surface-rendering, operative photo, or operative drawing), this must be “pixels”. See section 3.4.1: Electrode locations for more information on electrode locations. |
```

Comment:
1) `ACPC` is never mentioned in Appendix VIII. 
2) It is also not clear if this field can take any value or those listed in Appendix VIII + `Other` + `pixels`. One way to clarify is to list allowed values.
3) [minor] why `Other` with capital 'O' and 'pixels` with lowercase?